examples/design/simple_reactor.py

```diff
@@ -0,0 +1,547 @@
+                "kappa_l": {"value": params.kappa_l.value},
+                "delta_u": {"value": params.delta_u.value},
+                "delta_l": {"value": params.delta_l.value},
+                "phi_u_pos": {"value": params.phi_pos_u.value, "lower_bound": 0.0},
+                "phi_u_neg": {"value": params.phi_neg_u.value, "lower_bound": 0.0},
+                "phi_l_pos": {"value": params.phi_pos_l.value, "lower_bound": 0.0},
+                "phi_l_neg": {
+                    "value": params.phi_neg_l.value,
+                    "lower_bound": 0.0,
+                    "upper_bound": 90,
+                },
+            }
+        )
+
+
+class PlasmaBuilder(Builder):
+    """
+    Our PlasmaBuilder
+    """
+
+    param_cls = None
+
+    def __init__(self, wire: BluemiraWire, build_config: Dict):
+        super().__init__(None, build_config)
+        self.wire = wire
+
+    def build(self) -> Plasma:
+        """
+        Run the full build of the Plasma
+        """
+        xz = self.build_xz()
+        return Plasma(
+            self.component_tree(
+                xz=[xz],
+                xy=[Component("")],
+                xyz=[self.build_xyz(xz.shape)],
+            )
+        )
+
+    def build_xz(self) -> PhysicalComponent:
+        """
+        Build the xz Component of the Plasma
+        """
+        component = PhysicalComponent("LCFS", BluemiraFace(self.wire))
+        component.display_cad_options.color = BLUE_PALETTE["PL"]
+        component.display_cad_options.transparency = 0.5
+        return component
+
+    def build_xyz(self, lcfs: BluemiraFace) -> PhysicalComponent:
+        """
+        Build the xyz Component of the Plasma
+        """
+        shape = revolve_shape(lcfs, degree=359)
+        component = PhysicalComponent("LCFS", shape)
+        component.display_cad_options.color = BLUE_PALETTE["PL"]
+        component.display_cad_options.transparency = 0.5
+        return component
+
+
+# %%[markdown]
+
+# And now the TF Coil, in this instance for simplicity we are only making one TF coil.
+# If more TF coils were to be required the build_xyz of `TFCoilBuilder` would need to
+# be modified.
+# Notice that only `TFCoilBuilder` has required parameters in this case.
+
+# %%
+
+
+class TFCoilDesigner(Designer):
+    """TF coil Designer"""
+
+    param_cls = None
+
+    def __init__(self, plasma_lcfs, params, build_config):
+        super().__init__(params, build_config)
+        self.lcfs = plasma_lcfs
+        self.parameterisation_cls = get_class_from_module(
+            self.build_config["param_class"],
+            default_module="bluemira.geometry.parameterisations",
+        )
+
+    def run(self) -> GeometryParameterisation:
+        """TF coil run method"""
+        parameterisation = self.parameterisation_cls(
+            var_dict={
+                "x1": {"value": 3.0, "fixed": True},
+                "x2": {"value": 15, "lower_bound": 12},
```

Comment:
Won't this only work for a `PrincetonD`? What happens if we pass in another geometry parameterisation? Isn't this `var_dict` usually defined in the `build_config`?